dask/dataframe/io/tests/test_parquet.py

```diff
@@ -1315,24 +1329,29 @@ def test_to_parquet_pyarrow_w_inconsistent_schema_by_partition_succeeds_w_manual
         }
     )
 
-    ddf = dd.from_pandas(df, npartitions=2)
-    schema = pa.schema(
-        [
-            ("arrays", pa.list_(pa.int64())),
-            ("strings", pa.string()),
-            ("tstamps", pa.timestamp("ns")),
-            ("tz_tstamps", pa.timestamp("ns", timezone)),
-            ("partition_column", pa.int64()),
-        ]
-    )
-    ddf.to_parquet(
-        str(tmpdir), engine="pyarrow", partition_on="partition_column", schema=schema
-    )
-    ddf_after_write = (
-        dd.read_parquet(str(tmpdir), engine="pyarrow", calculate_divisions=False)
-        .compute()
-        .reset_index(drop=True)
-    )
+    with dask.config.set({"dataframe.convert_string": False}):
```

Comment:
Thoughts on replacing the `dask.config.set` here (and in other similar contexts) with a `pytest` `xfail` marker? 

```python
xfail_with_pyarrow_string = pytest.mark.xfail(dask.config.get("dataframe.convert_string"), reason="Known failure with pyarrow strings")

...

@xfail_with_pyarrow_string
@PYARROW_MARK
def test_to_parquet_pyarrow_w_inconsistent_schema_by_partition_succeeds_w_manual_schema(
    tmpdir,
):
```

If we do something like this, we should also make sure `xfail_with_pyarrow_string` is defined in some central location

Reply:
Good idea.

Reply:
Defined `xfail_with_pyarrow_string` in `dask.tests`.